### PR TITLE
Announcement push notifications

### PR DESCRIPTION
**Pull request by steam**
This PR does not yet handle loading in announcement notifications. We'll need to add that in once the announcement stuff is baked.

### CODE REVIEW

**Comment by jayzes** [comment]
Do we need to add the setting to the store/schema too?

**Comment by jayzes** [comment]
(cc @justin-holmes)

**Comment by steam** [comment]
Great catch @jayzes, we definitely do.

**Comment by steam** [comment]
@jayzes / @Justin-Holmes - Is the announcement push notification PR up on any staging servers for me to verify? I'm checking for the `notify_of_annnouncements_via_push` property in https://github.com/ello/ello/pull/1897/files#diff-39382771bce66dbb2411e857dfe2c7be

**Comment by Justin-Holmes** [comment]
@steam Pushing that up at to stage2 right now, so it should be up in a few.

File: Specs/Networking/ElloURISpec.swift [hunk 1]
```diff
@@ -179,6 +180,7 @@ class ElloURISpec: QuickSpec {
                     }
                 }
 
+"notifications/all"
```
Comment (colinta) [inline Specs/Networking/ElloURISpec.swift]: This is a funny little string. Hey string! How are ya? :-D

**Comment by steam** [comment]
Turns out none of this is needed. We changed the structure of the announcement push notifications server side. 🎉 

No code is the best code.

**Comment by steam** [comment]
**_JUUUUUUUST Kidding_**. We do need the settings for announcement push notifications. Thankfully @jayzes remembers this, over and over again. Because I certainly don't for some reason. The PR **_should_** be good to go now.

**Comment by jayzes** [comment]
~No~Some code is the best code.

**Pull request merged**